src/nbl/CMakeLists.txt

```diff
@@ -576,132 +576,129 @@ if(_NBL_COMPILE_WITH_GLI_)
 endif()
 
 # DXC
-if(_NBL_COMPILE_WITH_DXC_)
-	add_dependencies(Nabla dxc)
-	target_include_directories(Nabla PRIVATE 
-		"${DXC_BINARY_DIR_GE}/tools/clang/tools/dxcompiler"
-		"${DXC_SOURCE_DIR}/tools/clang/tools/dxcompiler"
-		"${DXC_SOURCE_DIR}/tools/clang/include"
-		"${DXC_BINARY_DIR_GE}/tools/clang/include"
-		"${DXC_SOURCE_DIR}/include/llvm/llvm_assert"
-		"${DXC_BINARY_DIR_GE}/include"
-		"${DXC_SOURCE_DIR}/include"
-		"${DXC_BINARY_DIR_GE}/include/dxc/Tracing"
-		"${DXC_BINARY_DIR_GE}/utils/version"
-	)
-	target_link_libraries(Nabla INTERFACE "${DXC_DXCOMPILER_LIB}")
+add_dependencies(Nabla dxc)
+target_include_directories(Nabla PRIVATE 
+	"${DXC_BINARY_DIR_GE}/tools/clang/tools/dxcompiler"
+	"${DXC_SOURCE_DIR}/tools/clang/tools/dxcompiler"
+	"${DXC_SOURCE_DIR}/tools/clang/include"
+	"${DXC_BINARY_DIR_GE}/tools/clang/include"
+	"${DXC_SOURCE_DIR}/include/llvm/llvm_assert"
+	"${DXC_BINARY_DIR_GE}/include"
+	"${DXC_SOURCE_DIR}/include"
+	"${DXC_BINARY_DIR_GE}/include/dxc/Tracing"
+	"${DXC_BINARY_DIR_GE}/utils/version"
```

Comment:
are you sure Nabla needs to know about all these?

Reply:
checked, we need to "inherit" those paths because they don't provide component based install target